<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsUi</class>
 <widget class="QWidget" name="OptionsUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>207</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>190</width>
        <height>399</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Values</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="scan_check">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Scan Disc/Image</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="scan_widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>23</number>
          </property>
          <item>
           <widget class="QLineEdit" name="scan_line">
            <property name="placeholderText">
             <string>Disc Name</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="disc_widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <weight>75</weight>
              <italic>true</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Disc Options</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="value_widget" native="true">
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Speed</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="speed_combo">
               <property name="editable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Copies Number</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="copies_spin">
               <property name="minimum">
                <number>1</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="on_fly_check">
            <property name="text">
             <string>Burn On fly</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="multi_session_check">
            <property name="text">
             <string>MultiSession</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="dummy_check">
            <property name="text">
             <string>Dummy Burn</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="eject_check">
            <property name="text">
             <string>Eject</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="image_widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <weight>75</weight>
              <italic>true</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Image Options</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="mount_check">
            <property name="text">
             <string>Mount Image</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="library_check">
            <property name="text">
             <string>Add To Library</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="library_widget" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="leftMargin">
              <number>23</number>
             </property>
             <item>
              <widget class="QLineEdit" name="library_name_line">
               <property name="placeholderText">
                <string>Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="library_tags_line">
               <property name="placeholderText">
                <string>Tags</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
